heft-plugins/heft-dev-cert-plugin/.npmignore

```diff
@@ -28,3 +28,4 @@
 #--------------------------------------------
 
 # (Add your project-specific overrides here)
+!heft-plugin.json
```

Comment:
Same comment as the Jest plugin PR - https://github.com/D4N14L/rushstack/pull/3#discussion_r910472285